app/components/chart-published.tsx

```diff
@@ -142,15 +140,15 @@ export const ChartPublishedInner = ({
             </HintRed>
           </Box>
         )}
-        {!isDataSourceTrusted && (
+        {/* {!isDataSourceTrusted && (
           <Box sx={{ mb: 4 }}>
             <HintBlue iconName="hintWarning">
               <Trans id="data.source.notTrusted">
                 This chart is not using a trusted data source.
               </Trans>
             </HintBlue>
           </Box>
-        )}
+        )} */}
```

Comment:
Why the comment ?

Reply:
Whoops, this shouldn't be commented out, thanks for catching! Same for the above comment.